src/casts/specify-representation.rkt

```diff
@@ -974,7 +976,7 @@ but a static single assignment is implicitly maintained.
            [other (error 'dyn-immediate-tag=? "expected type literal: ~a" t)])]
         ;; Observable Results Representation
         [(Blame (app recur e))
-         (begin$ (op$ Print e) (op$ Exit (Quote -1)) UNDEF-IMDT)]
+         (begin$ (op$ Print e) (op$ Exit EXIT-FAILURE) UNDEF-IMDT)]
```

Comment:
We should move these messages to stderr. The easiest solution would be to print with eprintf, but perhaps it is time to add input, output, and io ports as a types.

Reply:
I think this is out of the scope of this PR, however I would recommend to open an issue for this.